service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/http/HttpRequest.java

```diff
@@ -36,6 +38,7 @@ public HttpRequest(URL url, HttpMethod method, byte[] body) throws IOException
         // Codes_SRS_SERVICE_SDK_JAVA_HTTPREQUEST_12_003: [The function shall use the given HTTPS method (i.e. GET) as the request method.]
         // Codes_SRS_SERVICE_SDK_JAVA_HTTPREQUEST_12_004: [If an IOException occurs in setting up the HTTPS connection, the function shall throw an IOException.]
         this.connection = new HttpConnection(url, method);
+        log.debug("Created connection: url={}, method={}, connection={}", url, method, connection);
```

Comment:
>connection [](start = 27, length = 10)

created request, not connection. connection is created during send()

Reply:
`connection` object is created on line 40. and `connection` will be a keyword to track this call.